packages/shylu/shylu_dd/frosch/test/TestLaplacian_Epetra/ParametersRGDSW.xml

```diff
@@ -48,7 +48,7 @@
         <ParameterList name="OneLevelOperator">
```

Comment:
I am missing something. I thought the new CoarseOperator is in addition to the old one. However, I see only the HarmonicCoarseOperator now. I also thought there will be a way to choose the Operator in the Parameter list. Is there one ?

Reply:
Maybe, my explanation was not precise: 
I basically introduced the HarmonicCoarseOperator which now contains part of the functionality of the GDSWCoarseOperator, such that I can next derive an alternative GDSWCoarseOperator which can also use the methods from the new HarmonicCoarseOperator.
This pull request is just an intermediate step for moving to the alternative GDSWCoarseOperator.